packages/bot/src/app.jsx

```diff
@@ -16,6 +17,11 @@ class App extends React.Component {
             </Provider>
         );
     }
+
+    // eslint-disable-next-line
```

Comment:
@aaron-binary what is the error here?

Reply:
`class-methods-use-this`: https://eslint.org/docs/rules/class-methods-use-this